airbyte-integrations/connectors/source-the-guardian-api/source_the_guardian_api/spec.yaml

```diff
@@ -0,0 +1,62 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/the-guardian-api
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: The Guardian Api Spec
+  type: object
+  required:
+    - api_key
+  additionalProperties: true
+  properties:
+    api_key:
+      title: API Key
+      type: string
+      description: Your API Key. See <a href="https://open-platform.theguardian.com/access/">here</a>. The key is case sensitive.
+      airbyte_secret: true
+    query:
+      title: Query
+      type: string
+      description: (Optional) The query (q) parameter filters the results to only those that include that search term. The q parameter supports AND, OR and NOT operators.
+      examples:
+        - environment AND NOT water
+        - environment AND political
+        - amusement park
+        - political
+    tag:
+      title: Tag
+      type: string
+      description: (Optional) A tag is a piece of data that is used by The Guardian to categorise content. Use this parameter to filter results by showing only the ones matching the entered tag. See <a href="https://content.guardianapis.com/tags?api-key=test">here</a> for a list of all tags, and <a href="https://open-platform.theguardian.com/documentation/tag">here</a> for the tags endpoint documentation.
+      examples:
+        - environment/recycling
+        - environment/plasticbags
+        - environment/energyefficiency
+    section:
+      title: Section
+      type: string
+      description: (Optional) Use this to filter the results by a particular section. See <a href="https://content.guardianapis.com/sections?api-key=test">here</a> for a list of all sections, and <a href="https://open-platform.theguardian.com/documentation/section">here</a> for the sections endpoint documentation.
+      examples:
+        - media
+        - technology
+        - housing-network
+    order-by:
+      title: Order By
+      type: string
+      description: (Optional) Use this to sort the results. The three available sorting options are - newest, oldest, relevance.
+      pattern: ^(newest|oldest|relevance)$
+      examples:
+        - newest
+        - oldest
+        - relevance
+    from-date:
+      title: From Date
+      type: string
+      description: (Optional) Use this to set the minimum date (YYYY-MM-DD) of the results. Results older than the from-date will not be shown.
+      pattern: ^([1-9][0-9]{3})\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])$
+      examples:
+        - YYYY-MM-DD
+    to-date:
+      title: To Date
+      type: string
+      description: (Optional) Use this to set the maximum date (YYYY-MM-DD) of the results. Results newer than the to-date will not be shown.
+      pattern: ^([1-9][0-9]{3})\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])$
+      examples:
+        - YYYY-MM-DD
```

Comment:
I saw the API support query using start date and end date, this could enable incremental syncs. If you use always the order-by in the API request as `oldest`. Check the tutorial: https://docs.airbyte.com/connector-development/config-based/tutorial/incremental-reads

Reply:
Also would be better to change the parameter `from-date` to `start_date` and `to-date` to `end_date` because is more usually in other connectors.

Reply:
Sure, I'll try to enable incremental syncs. Thank you providing the tutorial! 
I'll also change the parameter names of `from-date` and `to-date`.

Reply:
@marcosmarxm I've added incremental sync and changed the parameter names as required! All acceptance tests are passing (see comment below for SS)